frontend/pages/hosts/details/HostDetailsPage/HostActionsDropdown/HostActionsDropdown.tsx

```diff
@@ -79,6 +79,10 @@ const HostActionsDropdown = ({
   // No options to render. Exit early
   if (options.length === 0) return null;
 
+  if (hostPlatform === "ios" || hostPlatform === "ipados") {
```

Comment:
Intentionally put hiding the dropdown within the HostActionsDropdown so 1. it can be tested in HostActionsDropdown.tests.tsx and 2) design discussions suggested there will be dropdown options in future iterations.